UI/HtmlControls/HtmlControls/HtmlMeter.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using Windows.Foundation;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Uno.Extensions;
+using Uno.UI.Runtime.WebAssembly;
+
+namespace HtmlControls
+{
+    [HtmlElement("meter")]
+    public partial class HtmlMeter : FrameworkElement
+    {
+        public HtmlMeter()
+        {
+            // Set a background to ensure pointer events are allowed
+            Background = new SolidColorBrush(Colors.Transparent);
```

Comment:
This is not supposed to be required anymore.

Reply:
I've removed this line